apps/web/app/(app)/environments/[environmentId]/integrations/page.tsx

```diff
@@ -14,12 +14,15 @@ import { getIntegrations } from "@formbricks/lib/integration/service";
 export default async function IntegrationsPage({ params }) {
   const environmentId = params.environmentId;
 
-  const [environment, integrations, userWebhooks, zapierWebhooks] = await Promise.all([
-    getEnvironment(environmentId),
-    getIntegrations(environmentId),
-    getCountOfWebhooksBasedOnSource(environmentId, "user"),
-    getCountOfWebhooksBasedOnSource(environmentId, "zapier"),
-  ]);
+  const [environment, integrations, userWebhooks, zapierWebhooks, makeWebhooks, n8nwebhooks] =
+    await Promise.all([
+      getEnvironment(environmentId),
+      getIntegrations(environmentId),
+      getCountOfWebhooksBasedOnSource(environmentId, "user"),
+      getCountOfWebhooksBasedOnSource(environmentId, "zapier"),
+      getCountOfWebhooksBasedOnSource(environmentId, "make"),
```

Comment:
please rename function to `getWebhookCountBySource` to follow the current naming pattern better